packages/swirl-components/src/components/swirl-inline-status/swirl-inline-status.css

```diff
@@ -25,7 +25,7 @@
   & .inline-status__icon {
     width: 1rem;
     height: 1rem;
-    margin-top: 0.125rem;
+    margin-top: 0.2rem;
```

Comment:
Why the odd number (3.2px)? The icon looks off now on my screen.

Reply:
This is how it looks to me:

Before - 0.125rem
<img width="550" alt="Screenshot 2024-05-21 at 08 14 25" src="https://github.com/getflip/swirl/assets/2084270/9858eff1-c5ce-4bec-bd2f-8d69ed6a3bee">

After - 0.2rem
<img width="567" alt="Screenshot 2024-05-21 at 08 11 28" src="https://github.com/getflip/swirl/assets/2084270/d6b54b62-0885-4fd3-9580-0a9fc85c6f2d">

I think this could be a matter of perception. Both me an Paulo thought the icon was a bit more to the top then it should.

This is how it looks on Figma:
<img width="854" alt="Screenshot 2024-05-21 at 08 30 46" src="https://github.com/getflip/swirl/assets/2084270/b4d08b14-0826-4676-9157-0204aab4b09a">

In theory the `0.125rem` = `2px` it's correct but it did not look good to me.

Can you provide a screenshot how it looks on your screen just for understanding?
Nevertheless, I happy to drop this change, that why I put it on separate commit and asked for special attention on the review. 😄

Reply:
Probably an issue with different browsers or screen pixel density (I'm on Chrome and a 5k monitor). It's more noticeable with size "s". Maybe `vertical-align` could help?

Before: 
<img width="196" alt="Bildschirmfoto 2024-05-21 um 10 49 57" src="https://github.com/getflip/swirl/assets/1019414/018890bd-62a3-4fdb-b212-eced5a857046">

After:
<img width="199" alt="Bildschirmfoto 2024-05-21 um 10 51 31" src="https://github.com/getflip/swirl/assets/1019414/59bf398c-4e5c-4f87-b48d-77cff4ad0daf">

Reply:
Ok I totally see that it looks worst on yours than mine. I will drop the commit.

Any kind on alignment not dependent on margins would be better, but in the limit the text can be too long and become multiline and them I think we still want to keep the alignment with the first line.

<img width="347" alt="Screenshot 2024-05-21 at 10 13 10" src="https://github.com/getflip/swirl/assets/2084270/29089daf-206c-4219-a616-278c0f436df3">

I can have a look at the alignment later, for me it's not a blocker now 😉